packages/www/docs/api-reference/ingest.mdx

```diff
@@ -15,14 +15,16 @@ Use `/ingest` to discover which ingest data center is closest. Ingest and
 playback URLs for each livestream session must be from the same region.
 
 ```bash
-curl "https://livepeer.com/api/ingest"`
+curl -H 'authorization: Bearer {api_key}' \
```

Comment:
I don't know if authorization needs a specific mention here, since there's a full page dedicated to it in docs: <https://livepeer.com/docs/api-reference/authentication>